README.md

```diff
@@ -108,14 +108,14 @@ class DemoHandler < RequestHandler::Base
     page do
       default_size 10
       max_size 20
-      comments do
+      resource :comments do
         default_size 20
         max_size 100
       end
     end
 
     include_options do
-      allowed Dry::Types["strict.string"].enum("comments", "author")
+      allowed Dry::Types::Strict::String.enum("comments", "author")
```

Comment:
does this even still work with dry 1.x ? referencing this https://github.com/runtastic/request_handler/pull/43/files#r347246151

Reply:
Changed by accident, no it doesn't. thanks!